src/python/pants/engine/target.py

```diff
@@ -703,14 +703,40 @@ def compute_value(
 
 
 class Tags(StringSequenceField):
+    """Arbitrary strings that you can use to describe a target.
+
+    For example, you may tag some test targets with 'integration_test' so that you could run
+    `./pants --tags='integration_test' test ::` to only run on targets with that tag.
+    """
+
     alias = "tags"
 
 
+# TODO(#9388): remove
```

Comment:
As an aside - this might be nice to keep / resurrect. It allows exploration of an existing repo via, say `./pants list`. I saw this most commonly used to highlight how a utility binary target might be used; ie:
```
$ ./pants list --documented src/java/org/pantsbuild::
src/java/org/pantsbuild/tools/junit:main
  A replacement for org.junit.runner.JUnitCore.main that adds:
    + support for running individual test methods using [classname]#[methodname]
    + support for stderr and stdout redirection for tests
    + support for ant style junit-report xml output
    + support for per test class timer
    + support for running test classes in parallel
src/java/org/pantsbuild/tools/runner:runner-binary
  A command line tool for fixing problems with custom classloading and synthetic jar.
```
```

Reply:
Cool. I'm okay with keeping it then. Thanks :)